src/lib/storage/index/adaptive_radix_tree/adaptive_radix_tree_nodes.cpp

```diff
@@ -205,8 +205,8 @@ AbstractIndex::Iterator ARTNode16::end() const {
 
 ARTNode48::ARTNode48(const std::vector<std::pair<uint8_t, std::shared_ptr<ARTNode>>>& children) {
   _index_to_child.fill(INVALID_INDEX);
-  for (uint8_t i = 0u; i < children.size(); ++i) {
-    _index_to_child[children[i].first] = i;
+  for (auto i = size_t{0}; i < children.size(); ++i) {
```

Comment:
ASS. 

Well ... not really. But maybey `child_id`?

Reply:
The whole ART is full of `i`s, some of which are child_ids, others are not. Once we start to use it, we should look at refactoring it.